lxd-migrate/utils.go

```diff
@@ -142,6 +142,13 @@ func transferRootfs(ctx context.Context, dst lxd.InstanceServer, op lxd.Operatio
 	return nil
 }
 
+func (c *cmdMigrate) connectLocal() (lxd.InstanceServer, error) {
+	args := lxd.ConnectionArgs{}
+	args.UserAgent = fmt.Sprintf("LXC-MIGRATE %s", version.Version)
```

Comment:
LXD-MIGRATE?

Reply:
At first I've set `LXD-MIGRATE`, but then realized that we already use `LXC-MIGRATE`:
https://github.com/canonical/lxd/blob/8d645a000d5bb57dd9b7d976341e0615e4281cbb/lxd-migrate/main_migrate.go#L125-L128

and

https://github.com/canonical/lxd/blob/8d645a000d5bb57dd9b7d976341e0615e4281cbb/lxd-migrate/utils.go#L192-L194

Reply:
That seems incorrect to me, suggest we change both places.

Reply:
Fixed